bench-vortex/src/bin/tpch.rs

```diff
@@ -469,11 +468,12 @@ async fn bench_main(
 
 fn verify_duckdb_tpch_results(scale_factor: u8, duckdb_path: PathBuf) -> anyhow::Result<()> {
     let query_dir = PathBuf::from("duckdb-vortex/duckdb/extension/tpch/dbgen/queries");
-    let tmp_dir = format!(
-        "{}/spiral-tpch",
+    let tempdir = temp_dir();
+    let dir = env::var("TMPDIR")
         // $RUNNER_TEMP is defined by GitHub Actions.
-        env::var("TMPDIR").unwrap_or(env::var("RUNNER_TEMP")?)
-    );
+        .and(env::var("RUNNER_TEMP"))
```

Comment:
`RUNNER_TEMP` is a fallback in case `TMPDIR` is not set.

Reply:
Not part of this PR, but we could revisit all tmpdir instances and replace them with a relative `data_dir` path.